doc/faq/customization.rst

```diff
@@ -85,6 +108,23 @@ A good stating point might be the themes at http://bootswatch.com.
 .. note:: If you add your own CSS file, the file *bootstrap-theme.css* will
    not be loaded anymore. So you might start with a copy of the original file.
 
+
+Use web server substitute module
```

Comment:
Hm, even though the solution using the substitute module works, it seems a bit hacky :) What do you think about adding a more "official" way to change the title in 3.1? I can add a issue for that.

Reply:
Yes, please do it.

Reply:
I've opened #1624.